google/cloud/sqlalchemy_spanner/requirements.py

```diff
@@ -68,6 +68,11 @@ def isolation_level(self):
     def sequences(self):
         return exclusions.closed()
 
+    @property
+    def temporary_tables(self):
+        """target database supports temporary tables"""
+        return exclusions.closed()
```

Comment:
Please also update the README to say that it's no longer supported.

Reply:
The README already [states](https://github.com/googleapis/python-spanner-sqlalchemy/blob/a64644b1a36fd10c57ddac5a15105d1a412ffbde/README.md?plain=1#L198):
```
Temporary tables are not supported, real tables are used instead.
```

Reply:
Oh right, I meant that we should just say "Temporary tables are not supported." so that users don't attempt to use this as a feature.